components/datadog/fakeintake/connectionExporter.go

```diff
@@ -9,37 +9,37 @@ import (
 )
 
 const (
-	stackKey = "fakeintake-url"
+	stackKey = "fakeintake-ip-address"
 )
 
 // ConnectionExporter contains pulumi side data and the export key
 type ConnectionExporter struct {
-	URL      pulumi.StringInput
+	Host     pulumi.StringInput
 	stackKey string
 }
 
 // ClientData client side data
 type ClientData struct {
-	URL string
+	Host string
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities. -->
**💭 thought**
This will require an update on the dd-agent side, I agree it is less misleading - fun fact I misled myself as I believed it was containing the protocol prefix

Reply:
@pducolin Here are the required changes on `datadog-agent` side: DataDog/datadog-agent#17566.